pennylane/qchem/hamiltonian.py

```diff
@@ -170,21 +175,31 @@ def _fermionic_hamiltonian(*args):
             *args (array[array[float]]): initial values of the differentiable parameters
 
         Returns:
-            tuple(array[float], list[list[int]]): the Hamiltonian coefficients and operators
+            Union[FermiSentence, tuple(array[float], list[list[int]])]: fermionic Hamiltonian
         """
+
         core_constant, one, two = electron_integrals(mol, core, active)(*args)
 
-        return fermionic_observable(core_constant, one, two, cutoff)
+        if not fs:
+            warnings.warn(
+                "This function will return a fermionic operator by default in the next release. For"
+                " details, see the Fermionic Operators tutorial:"
+                " https://pennylane.ai/qml/demos/tutorial_fermionic_operators."
+                " Currently, a fermionic operator can be returned by setting the `fs` kwarg to `True`."
+            )
+        return fermionic_observable(core_constant, one, two, cutoff, fs)
 
     return _fermionic_hamiltonian
 
 
-def diff_hamiltonian(mol, cutoff=1.0e-12, core=None, active=None):
+def diff_hamiltonian(mol, cutoff=1.0e-12, core=None, active=None, fs=True):
     r"""Return a function that computes the qubit Hamiltonian.
 
     Args:
         mol (~qchem.molecule.Molecule): the molecule object
         cutoff (float): cutoff value for discarding the negligible electronic integrals
+        core (list[int]): indices of the core orbitals
+        active (list[int]): indices of the active orbitals
```

Comment:
Missing doc-string

If the results of using a fermi sentence internally produces identical results, then we don't need the keyword argument? 

```suggestion
        core (list[int]): indices of the core orbitals
        active (list[int]): indices of the active orbitals
        fs (bool): if True, a fermi sentence is used to construct the mapping
```